ecosystem/web-wallet/src/core/components/NetworkBody.tsx

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  VStack,
+  Flex,
+  SimpleGrid,
+  Heading,
+  useRadioGroup,
+} from '@chakra-ui/react';
+import React from 'react';
+import { AptosNetwork } from 'core/utils/network';
+import { DEVNET_NODE_URL, LOCAL_NODE_URL } from 'core/constants';
+import useWalletState from 'core/hooks/useWalletState';
+import { useQuery } from 'react-query';
+import { getLocalhostIsLive } from 'core/queries/network';
+import useSwitchNetwork from 'core/mutations/network';
+import NetworkListItem from './NetworkListItem';
+
+interface NetworkPreference {
+  description?: string;
+  title?: 'Mainnet' | 'Testnet' | 'Devnet' | 'Localhost';
+  value: AptosNetwork;
+}
+
+const networkPreferences: NetworkPreference[] = [
+  {
+    title: 'Devnet',
+    value: DEVNET_NODE_URL,
+  },
+  {
+    title: 'Localhost',
+    value: LOCAL_NODE_URL,
+  },
+];
+
+export default function NetworkBody() {
+  const {
+    aptosNetwork,
+  } = useWalletState();
+  const { data: testnetIsLive } = useQuery('getTestnetStatus', getLocalhostIsLive, { refetchInterval: 1000 });
```

Comment:
Why is it `testnetIsLive` when the query is `getLocalhostIsLive`?

Reply:
It's local testnet, i will fix the wording